sentry-android-replay/src/main/java/io/sentry/android/replay/capture/SessionCaptureStrategy.kt

```diff
@@ -140,17 +141,26 @@ internal class SessionCaptureStrategy(
     override fun convert(): CaptureStrategy = this
 
     private fun createCurrentSegment(taskName: String, onSegmentCreated: (ReplaySegment) -> Unit) {
-        val now = dateProvider.currentTimeMillis
-        val currentSegmentTimestamp = segmentTimestamp ?: return
-        val segmentId = currentSegment
-        val duration = now - currentSegmentTimestamp.time
-        val replayId = currentReplayId
-        val height = recorderConfig.recordingHeight
-        val width = recorderConfig.recordingWidth
-        replayExecutor.submitSafely(options, "$TAG.$taskName") {
-            val segment =
-                createSegmentInternal(duration, currentSegmentTimestamp, replayId, segmentId, height, width)
-            onSegmentCreated(segment)
+        recorderConfig?.let { config ->
+            val now = dateProvider.currentTimeMillis
+            val currentSegmentTimestamp = segmentTimestamp ?: return
+            val segmentId = currentSegment
+            val duration = now - currentSegmentTimestamp.time
+            val replayId = currentReplayId
+            replayExecutor.submitSafely(options, "$TAG.$taskName") {
+                val segment =
+                    createSegmentInternal(
+                        duration,
+                        currentSegmentTimestamp,
+                        replayId,
+                        segmentId,
+                        config.recordingHeight,
+                        config.recordingWidth,
+                        config.frameRate,
+                        config.bitRate
+                    )
+                onSegmentCreated(segment)
+            }
```

Comment:
can we add a log maybe saying that recorderConfig isn't available that's why we're not capturing? Would help us investigate issues.